vncdotool/client.py

```diff
@@ -251,11 +251,11 @@ def mouseUp(self: TClient, button: int) -> TClient:
 
         return self
 
-    def captureScreen(self, filename: str, incremental: bool = False) -> Deferred:
+    def captureScreen(self, filename: str, incremental: bool = False, format=None) -> Deferred:
```

Comment:
Please add PEP-484 type hints:
```suggestion
    def captureScreen(self, filename: str, incremental: bool = False, format: Optional[str] = None) -> Deferred:
```